src/test/java/codeu/controller/LoginServletTest.java

```diff
@@ -103,6 +106,6 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.verify(mockSession).setAttribute("user", "test username");
     Mockito.verify(mockResponse).sendRedirect("/conversations");
     //check if password is correct
-//    Mockito.verify("password", Mockito.getPassword());
-  }
+    //Mockito.verify("password", Mockito.getPassword());
```

Comment:
Why do these tests need to be commented out?

Reply:
These tests do not pass after the register servlet is added.
I can try to see what needs to be changed.

Reply:
Project 1-7 modify loginservlet
This change will now cause the LoginServletTest to fail. Fix that test before continuing. Remove tests that are no longer needed, and add new tests that check the different kinds of input the LoginServlet can receive.

Reply:
Okay so I read through the tests and before the login servlet is also performing functions of register servlets, and that's what these tests are for. Now register test handles that so these tests are not needed anymore.